src/traces/sunburst/attributes.js

```diff
@@ -215,5 +215,18 @@ module.exports = {
     },
     sort: pieAttrs.sort,
 
+    root: {
```

Comment:
This has no effect on `sunburst` traces.
So, let's only add it to the `treemap`.

Reply:
In case the root node of a sunburst trace is not "", a "bull's eye" is shown in the middle of the sunburst chart, and the new attributes colors that eye. Actual version of plotly sets the color of the bull's eyehardcoded to rgba(0,0,0,0).

[see fiddle](https://jsfiddle.net/thierryve/638g7c9r/)

`root: { color: 'yellow'}`